src/Persistence.Abstractions/Persistence.Abstractions.csproj

```diff
@@ -9,10 +8,24 @@
 		<PackageId>Nventive.Persistence.Abstractions</PackageId>
 		<Description>Nventive.Persistence.Abstractions</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>persistence;uwp;ios;android</PackageTags>
```

Comment:
Abstactions is used by Uno and Uno.WinUI if I'm not mistaken, shouldn't we include WinUI as well here?